common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -193,6 +193,10 @@ def modulestore():
             settings.MODULESTORE['default'].get('OPTIONS', {})
         )
 
+        if settings.FEATURES.get('CUSTOM_COURSES_EDX'):
+            from ccx.modulestore import CCXModulestoreWrapper
```

Comment:
I restored this after changing it as suggested because it causes circular import errors when the import is at module level.  We must leave it like this, I guess.